src/middlewared/middlewared/plugins/smb.py

```diff
@@ -475,22 +476,22 @@ def getparm(self, parm, section):
 
     @accepts(Dict(
         'smb_update',
-        Str('netbiosname'),
-        Str('netbiosname_b'),
+        Str('netbiosname', validators=[Range(max=120)]),
+        Str('netbiosname_b', validators=[Range(max=120)]),
         List('netbiosalias', default=[]),
```

Comment:
Am I the only one who is annoyed with this newly introduced noise?
`netbiosname` is up to 120 characters, but `netbiosalias` can be of any length, is this what this code says?

Reply:
The model does enforce a restriction on this ( or tries to being more accurate ), but as we have it as a list, i wasn't sure what limit to impose here.

Reply:
@sonicaj Microsoft doesn't allow more than 15 characters for NetBIOS names and 255 characters for the FQDN.